connectors/src/connectors/slack/index.ts

```diff
@@ -278,6 +312,16 @@ export class SlackConnectorManager extends BaseConnectorManager<SlackConfigurati
       );
     }
 
+    // Resync webhook router configuration to remove deleted connector entry
```

Comment:
I envisioned it more to be the same code path as is the API, where it would always sync to the state that's in the DB, rather than having to get the list in the right state here. That would also mean calling it after the connector.delete(). But maybe I didn't organize the existing code in a way that makes that easy enough.